src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -20,16 +20,66 @@
 							name="auto-icon-on"
 						/>
 					</template>
+
 					<template v-slot:content>
-						<p>You'll be asked for a verification code when accessing you Kiva account.</p>
+						<div v-if="!isMfaActive">
+							<h2>{{ cardTitle }}</h2>
```

Comment:
What is the h2 if `isMfaActive` is true?

Reply:
I had that handled on line 54.
But your comment called out the duplication and I've simplified it now.